store/postgres/src/subgraph_store.rs

```diff
@@ -1264,6 +1264,16 @@ impl SubgraphStoreTrait for SubgraphStore {
         Ok(writable)
     }
 
+    async fn stop_subgraph(&self, loc: &DeploymentLocator) -> Result<(), StoreError> {
+        // Remove the writable from the cache and stop it
+        let deployment = loc.id.into();
+        let writable = self.writables.lock().unwrap().remove(&deployment);
```

Comment:
If I got it right [docs](https://doc.rust-lang.org/std/sync/struct.Mutex.html#method.lock) right, this would only panic due to a programming error, so it is relatively safe to unwrap here.
Is that correct?

Reply:
It would panic if the lock is poisoned, i.e., if some other thread paniced while holding the lock. In that case, it's anybody's guess what's happening and panicing here is really the only thing we can do.